pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1912,6 +1913,26 @@ func triggerPublishLocationToDest(ctxPtr *zedagentContext, dest destinationBitse
 	ctxPtr.triggerLocationInfo <- dest
 }
 
+func triggerPublishObjectInfo(ctxPtr *zedagentContext, infoType info.ZInfoTypes,
+	objectKey string) {
+	ctxPtr.triggerObjectInfo <- infoForObjectKey{
```

Comment:
How much buffering on this channel? Can we potentially hang due to doing a triggerPublish* from the main event loop, and only code in the main event loop will read the channel? Or is this triggerObjectInfo read by a separate go routine?

Reply:
I see it is read by a separate goroutine, so how about documenting that here in the two writers to the channel?

Reply:
Added comments for these functions.